### PR TITLE
Add unit test for logger

### PR DESCRIPTION
**Pull request by acltang**
closes https://github.com/hwsc-org/hwsc-file-transaction-svc/issues/24

Not sure how to test functions with a variable number of arguments (`*args`). 
The `exception()` function prints the stack trace only if it is called inside an `except:` block. Not sure how to unit test if a stack trace is being printed (I know it is working from manual tests).

### CODE REVIEW

File: logger/logger.py [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
 
+# Set up StringIO handler for unit test
+root = logging.getLogger()
+root.setLevel(logging.DEBUG)
+log_stream = StringIO()
+handler = logging.StreamHandler(log_stream)
+handler.setLevel(logging.DEBUG)
+handler.setFormatter(logging.Formatter("%(asctime)s [%(levelname)s] %(message)s", datefmt="%y/%m/%d %H:%M:%S"))
+root.addHandler(handler)
+
```
Comment (faraonc) [inline logger/logger.py]: Can you add context why these changes are needed?

Comment (acltang) [reply logger/logger.py]: The functions in `logger.py` do not return any values, they simply output to console. In order to capture this output with `pytest`, the output needs to be in either `stdout` or `stderr` streams. This code creates an identical `StringIO()` logging stream and attaches it to the main logger so that `test_logger.py` has a chance to access the contents of the output through `log_stream`. I cannot access the default logging `stdout` stream because it is automatically emptied after outputting to console. I am essentially duplicating the output of the logger and saving it because the original output of the logger cannot be accessed.

There is probably a better way to do this, but this is the only working solution I can come up with from my research. I may be missing an important concept entirely.
